packages/roosterjs-editor-api/lib/format/clearFormat.ts

```diff
@@ -1,11 +1,25 @@
 import execFormatWithUndo from './execFormatWithUndo';
-import { Editor } from 'roosterjs-editor-core';
+import { Editor, browserData } from 'roosterjs-editor-core';
+import setFontName from './setFontName';
+import setFontSize from './setFontSize';
+import setTextColor from './setTextColor';
+import setBackgroundColor from './setBackgroundColor';
 
 export default function clearFormat(editor: Editor): void {
     editor.focus();
     // We have no way if this clear format will really result in any DOM change
     // Let's just do it with undo
     execFormatWithUndo(editor, () => {
-        editor.getDocument().execCommand('removeFormat', false, null);
+        editor.runWithoutAddingUndoSnapshot(() => {
+            editor.getDocument().execCommand('removeFormat', false, null);
+
+            if (browserData.isEdge || browserData.isIE) {
```

Comment:
Only for IE and Edge? How about other browsers?

Reply:
Other browsers can already handle this well.